website/package.json

```diff
@@ -5,11 +5,13 @@
   "author": "HashiCorp",
   "dependencies": {
     "@hashicorp/mktg-global-styles": "2.1.0",
-    "@hashicorp/nextjs-scripts": "16.0.1",
+    "@hashicorp/nextjs-scripts": "16.2.0",
     "@hashicorp/react-button": "4.0.0",
-    "@hashicorp/react-docs-page": "^10.3.2",
+    "@hashicorp/react-docs-page": "10.9.4-alpha.18",
```

Comment:
I'm using a pre-release of `react-docs-page` at the moment.

We'll aim to cut a stable release before merging this PR, though it will likely be functionally identical to the pre-release, so if time is of the essence, we may choose to merge with the pre-release to unblock other work, and then quickly follow up to upgrade to a stable release.

- [ ] @zchsh Cut stable pre-release of `@hashicorp/react-docs-page`, update in deps

Reply:
No issues here. This will be released onto master and not pushed to production today so there is time. As long as we can render via preview on https://packer-git-master-hashicorp.vercel.app/ we are good to go and can follow up with an official release when ready.